app/Resources/views/themes/tastefulplanet/fields/ezrichtext/custom_tags/ezfacebook.html.twig

```diff
@@ -0,0 +1,14 @@
+<script>(function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = 'https://connect.facebook.net/en_EN/sdk.js#xfbml=1&version=v2.11';
+        fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));
+</script>
+
+<div
```

Comment:
I prefer this way: 
```twig
<div 
    class="fb-post" 
    style="display: {{ params.display }};" 
    data-href="{{ params.post_url }}" 
    data-width="{{ params.width ? params.width : '' }}">
</div>
```

Reply:
Fixed in 4014bf5.

Reply:
And amended in 5b17535. Hope this time It's actually well formed.